app/display/editor/src/main/java/org/csstudio/display/builder/editor/properties/PropertyPanelSection.java

```diff
@@ -29,20 +31,7 @@
 import org.csstudio.display.builder.model.WidgetProperty;
 import org.csstudio.display.builder.model.WidgetPropertyCategory;
 import org.csstudio.display.builder.model.persist.WidgetClassesService;
-import org.csstudio.display.builder.model.properties.ActionsWidgetProperty;
-import org.csstudio.display.builder.model.properties.BooleanWidgetProperty;
-import org.csstudio.display.builder.model.properties.ColorMapWidgetProperty;
-import org.csstudio.display.builder.model.properties.ColorWidgetProperty;
-import org.csstudio.display.builder.model.properties.CommonWidgetProperties;
-import org.csstudio.display.builder.model.properties.EnumWidgetProperty;
-import org.csstudio.display.builder.model.properties.FilenameWidgetProperty;
-import org.csstudio.display.builder.model.properties.FontWidgetProperty;
-import org.csstudio.display.builder.model.properties.MacrosWidgetProperty;
-import org.csstudio.display.builder.model.properties.PVNameWidgetProperty;
-import org.csstudio.display.builder.model.properties.PointsWidgetProperty;
-import org.csstudio.display.builder.model.properties.RulesWidgetProperty;
-import org.csstudio.display.builder.model.properties.ScriptsWidgetProperty;
-import org.csstudio.display.builder.model.properties.WidgetClassProperty;
+import org.csstudio.display.builder.model.properties.*;
```

Comment:
Minor issue: Can you revert the wildcard `import ...*` back to the specific imports?
Some IDEs will change over to wildcard imports after a certain number of imports from the same package, but we'd like to keep all imports explicit to avoid confusion down the road (https://github.com/ControlSystemStudio/phoebus/pull/2440#issuecomment-1299034799)

Reply:
Sure. I have changed the wildcard import statement to individual import statements now.